sdk/core/core-lro/src/lroEngine/requestUtils.ts

```diff
@@ -41,7 +41,7 @@ export function inferLroMode(
       resourceLocation:
         requestMethod === "PUT"
           ? requestPath
-          : requestMethod === "POST"
+          : ["POST", "PATCH"].includes(requestMethod)
```

Comment:
Would this be equivalent to making getLocation the default behavior if the request method is not `PUT` or `DELETE`. As I read the LRO diagram we'd like to go there when coming from any method but `PUT` or `DELETE`.

![image](https://user-images.githubusercontent.com/21109913/143928164-b97a3f0f-9004-4d9a-b87c-7aff0db4dc42.png)

Reply:
Yeah I suppose we can rewrite the check as `!["DELETE", "PUT"].includes(requestMethod)` but looking at the list of [http verbs list](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), I do not think we need to worry about the others (trace, connect, head, get, options).